app/src/main/java/com/tma/sparking/services/handler/ParkingFieldManager.java

```diff
@@ -0,0 +1,56 @@
+package com.tma.sparking.services.handler;
+
+import android.content.Context;
+import android.os.Handler;
+import android.os.Message;
+
+import com.tma.sparking.models.ParkingField;
+import com.tma.sparking.services.ParkingFieldService;
+import com.tma.sparking.services.http.Parking;
+
+import java.util.List;
+
+public class ParkingFieldManager {
```

Comment:
I think we can stop handler thread before activity stopped in onStop() method by calling HandlerThread.quit()